plugins/inputs/nfsclient/nfsclient.go

```diff
@@ -0,0 +1,458 @@
+package nfsclient
+
+import (
+	"bufio"
+	"log"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal/choice"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type NFSClient struct {
+	Fullstat          bool
+	IncludeMounts     []string
+	ExcludeMounts     []string
+	IncludeOperations []string
+	ExcludeOperations []string
+	Log               telegraf.Logger
```

Comment:
Trying to support you a bit:
```suggestion
	Fullstat          bool `toml:"fullstat"`
	IncludeMounts     []string `toml:"include_mounts"`
	ExcludeMounts     []string `toml:"exclude_mounts"`
	IncludeOperations []string `toml:"include_operations"`
	ExcludeOperations []string `toml:"exclude_operations"`
	Log               telegraf.Logger `toml:"-"`
```
Please remember to fix the formatting.

Reply:
That looks like magic markup/documentation.  That's that do?

Reply:
These are [struct tags](https://golangbyexample.com/struct-field-meta-or-tags/) that help the [toml parser](https://github.com/influxdata/toml) to figure out which config option (that's the tag) to map to which field. It allows to e.g. change the struct-field name independent of the option. This way it's also easy to trace a config-option's use in the code.

Reply:
Please mark as resolved.